pipewire/qubes-pw-module.c

```diff
@@ -1061,23 +1074,27 @@ int pipewire__module_init(struct pw_impl_module *module, const char *args)
             goto error;
         impl->domid = (uint16_t)domid;
 
-        size_t read_min = 1 << 15, write_min = 1 << 15;
+        size_t read_min = SIZE_MAX, write_min = SIZE_MAX;
         const char *record_size = pw_properties_get(props, QUBES_PW_KEY_RECORD_BUFFER_SPACE);
         const char *playback_size = pw_properties_get(props, QUBES_PW_KEY_PLAYBACK_BUFFER_SPACE);
         if (!record_size || !playback_size) {
             const char *buffer_size = pw_properties_get(props, QUBES_PW_KEY_BUFFER_SPACE);
-            if (buffer_size && !record_size)
+            if (!buffer_size) { // should always be set
+                pw_log_error("Packaging bug: %s not specified in configuration file",
```

Comment:
I don't think that's a good idea, better to always have some built-in default.

Reply:
Will do, especially since PipeWire makes it difficult for a configuration file’s default to be overridden by the user, yet still get picked up by this module.